cmd/bnsd/x/nft/username/handler_test.go

```diff
@@ -77,21 +77,28 @@ func TestHandleIssueTokenMsg(t *testing.T) {
 			expCheckError:   true,
 			expDeliverError: true,
 		},
-		{ // name already used
+		"name already used": {
 			owner:           alice.Address(),
 			id:              []byte("existing@example.com"),
 			details:         username.TokenDetails{Addresses: myNewChainAddresses},
 			expCheckError:   false,
 			expDeliverError: true,
 		},
-		{ // address already used
-			owner:           alice.Address(),
-			id:              []byte("any@example.com"),
-			details:         username.TokenDetails{Addresses: []username.ChainAddress{{BlockchainID: []byte("myNet"), Address: "bobsChainAddress"}}},
+		"address can have any number of names assigned": {
+			owner: alice.Address(),
+			id:    []byte("any@example.com"),
+			details: username.TokenDetails{
+				Addresses: []username.ChainAddress{
```

Comment:
Yes, this is the same address as above under a different name.
To make this cleaner, maybe define:

```
bobsAddress := username.ChainAddress{
    BlockchainID: []byte("myNet"),
    Address:      "bobsChainAddress",
}
```

then when refering to it in the test, it is clear to be the same as existing data (not some typo)